detect_variants/combine_docm.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "CombineVariants (GATK 3.6)"
+baseCommand: ["/usr/bin/java", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "CombineVariants"]
+arguments:
```

Comment:
I would define at minimum a 4GB RAM resource requirement. On full exome data sets, this often failed without a minimum RAM requirement. It should be equal to or greater than the heap size above.